{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/commands.py

```diff
@@ -12,11 +12,21 @@
 
 
 @click.command()
-def test():
+@click.option(
+    "-c",
+    "--coverage",
+    default=False,
```

Comment:
Thoughts on setting this to `True` by default?

Reply:
I change this to `True` by default. I would like to keep the behavior look like before adding this param, so set it to `False`